activesupport/lib/active_support/cache.rb

```diff
@@ -845,9 +845,7 @@ def expires_at=(value)
         end
       end
 
-      # Returns the size of the cached value. This could be less than
-      # <tt>value.bytesize</tt> if the data is compressed.
-      def bytesize
+      def bytesize # :nodoc:
```

Comment:
The entire `Entry` is nodoc, we can remove this from here, and we can keep the documentation.

Reply:
👍 https://github.com/rails/rails/commit/93e53886539fc837091bebe1a9071b93b2274635